### PR TITLE
Add new mq client. Adapt parser and consumer to new mq client.

### PR DESCRIPTION
**Pull request by madcake**

### CODE REVIEW

File: cmd/consumer/main.go [hunk 1]
```diff
@@ -2,19 +2,18 @@ package main
...
+	"time"
+
+	"github.com/trustwallet/blockatlas/services/notifier"
 
 	log "github.com/sirupsen/logrus"
+	"github.com/trustwallet/blockatlas/config"
 	"github.com/trustwallet/blockatlas/db"
 	"github.com/trustwallet/blockatlas/internal"
-	"github.com/trustwallet/blockatlas/mq"
+	"github.com/trustwallet/blockatlas/new_mq"
```
Comment (hewigovens) [inline cmd/consumer/main.go:16]: how about `mqueue`? sounds better than `new_`

**Review by hewigovens — APPROVED** [review]
LGTM, probably need to test it a bit before production

**Comment by vikmeup** [comment]
> LGTM, probably need to test it a bit before production

It's work in progress, almost done. We have done the testing in different phases, connect/disconnect mq/postres to see how it reconnect and continues the flow.

File: cmd/consumer/main.go (hunk 1 above)
Comment (vikmeup) [reply cmd/consumer/main.go:16]: It will be just `mq`, `new_mq` will become `mq` as there is no need to keep the old one

**Pull request closed without merging**